File: FilATours_HTML5/js/busstop.js
/* -*- Mode: js; js-indent-level: 2; indent-tabs-mode: nil -*- */
/* vim: set shiftwidth=2 tabstop=2 autoindent cindent expandtab nospell: */

'use strict';

var BusStop = function(name, city, lat, lon) {
  this._name = name;
  this._city = city;
  this._latitude = lat;
  this._longitude = lon;
  this.init();
};

BusStop.prototype.init = function() {
};

function InvalidBusStopException () {

}

File: BusTours/src/org/gitorious/scrapfilbleu/android/BusStop.java
/* vim: set ts=4 sw=4 et: */

package org.gitorious.scrapfilbleu.android;

public class BusStop {
    public String name;
    public String city;
    public Double lat;
    public Double lon;
    public Double dist;

    public BusStop(String name, String city, Double lat, Double lon, Double dist) {
        this.name = name;
        this.city = city;
        this.lat  = lat;
        this.lon  = lon;
        this.dist = dist;
    }
}
